scripts/repoint_frontend.py

```diff
@@ -0,0 +1,198 @@
+
+    parser = argparse.ArgumentParser(description="Script to modify the target.xml file based on the environment argument.")  # noqa: E501
+
+    parser.add_argument('shortcode', type=str, help="Environment identifier (e.g. 'gith1')")
+    args = parser.parse_args()
+
+    # modify target.xml
+    file_path = 'proxies/live/apiproxy/targets/target.xml'
+    modify_target_file(args.shortcode, file_path)
+
+    # modify AssignMessage.MessageBatches.Create.Request.xml
+    file_path = 'proxies/shared/policies/AssignMessage.MessageBatches.Create.Request.xml'
+    block_to_follow = [
+        '<AssignVariable>',
+        '<Name>requestpath</Name>',
+        '<Value>/api/v1/send</Value>',
+        '</AssignVariable>'
+    ]
+    lines_to_add = [
+        '    {{% if ENVIRONMENT_TYPE != \'sandbox\' %}}',
+        '        <AssignVariable>',
+        '            <Name>target.url</Name>',
+        '            <Value>https://comms-apim.de-{shortcode}.communications.national.nhs.uk/api/v1/send</Value>',
+        '        </AssignVariable>',
+        '    {% endif %}'
+    ]
+    add_lines_after_block(args.shortcode, file_path, block_to_follow, lines_to_add)
+
+    # modify proxies/shared/policies/AssignMessage.Messages.Create.Request.xml
+    file_path = 'proxies/shared/policies/AssignMessage.Messages.Create.Request.xml'
+    block_to_follow = [
+        '<AssignVariable>',
+        '<Name>requestpath</Name>',
+        '<Value>/api/v1/messages</Value>',
+        '</AssignVariable>'
+    ]
+    lines_to_add = [
+        '    {% if ENVIRONMENT_TYPE != \'sandbox\' %}',
+        '        <AssignVariable>',
+        '            <Name>target.url</Name>',
+        '            <Value>https://comms-apim.de-{shortcode}.communications.national.nhs.uk/api/v1/messages</Value>',
+        '        </AssignVariable>',
+        '    {% endif %}'
+    ]
+    add_lines_after_block(args.shortcode, file_path, block_to_follow, lines_to_add)
+
+    # modify proxies/shared/policies/AssignMessage.Messages.GetSingle.Request.xml
+    file_path = 'proxies/shared/policies/AssignMessage.Messages.GetSingle.Request.xml'
+    block_to_follow = [
+        '<AssignVariable>',
+        '<Name>requestpath</Name>',
+        '<Template>/api/v1/messages/{data.messageId}</Template>',
+        '</AssignVariable>'
+    ]
+    lines_to_add = [
+        '    {% if ENVIRONMENT_TYPE != \'sandbox\' %}',
+        '        <AssignVariable>',
+        '            <Name>target.url</Name>',
+        '            <Value>https://comms-apim.de-{shortcode}.communications.national.nhs.uk/api/v1/messages/{{data.messageId}}</Value>',  # noqa: E501
+        '        </AssignVariable>',
+        '    {% endif %}'
+    ]
+    add_lines_after_block(args.shortcode, file_path, block_to_follow, lines_to_add)
+
+    # modify proxies/shared/policies/AssignMessage.NhsAppAccounts.Get.Request.xml
+    file_path = "proxies/shared/policies/AssignMessage.NhsAppAccounts.Get.Request.xml"
+    block_to_follow = [
```

Comment:
Nit: Similarly these could also be constants, I'd rather declutter `if __name__ == "__main__"` of any const declaration so it's easier to see what is going on in terms of calling to methods